src/fromager/settings.py

```diff
@@ -93,10 +93,12 @@ def build_dir(self, pkg: str, sdist_root_dir: pathlib.Path) -> pathlib.Path:
             return sdist_root_dir / relative_build_dir
         return sdist_root_dir
 
-    def build_tag(self, pkg: str, version: Version | str) -> int:
+    def build_tag(self, pkg: str, version: Version | str) -> BuildTag:
         p = self.get_package_settings(pkg)
         changelog = p.get("changelog", {}).get(str(version), [])
-        return len(changelog)
+        if not changelog:
+            return ()
```

Comment:
Maybe the caller would want this to be `(0, "")` to make unpacking the tuple consistent? Or is an empty tuple recognized by everything that wants the BuildTag?

Reply:
yep empty tuple is recognized by everything that wants the BuildTag. This keeps the types consistent and we anyways to perform a check to ensure the tuple is not empty anywhere we use this method. If we don't then mypy complains about it

Reply:
Makes sense.